File: 0001 Code Chef/0021 Biryani classes/implementation/0011 lua/biryani_classes.lua
test = tonumber(io.read())
while(test > 0) do
    string_input = io.read()
    string_array = {}
    count = 1;
    for i in string.gmatch(string_input,"%S+") do
        string_array[count] = i
        count = count + 1
    end

    x = tonumber(string_array[1]);
    y = tonumber(string_array[2]);
    total = x * y
    print(total)
    test = test - 1
end

File: 0001 Code Chef/0021 Biryani classes/implementation/0009 dart/biryani_classes.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? test = int.tryParse(stringInput);
    if(test != null){
      int count = test;
      while(count > 0){
        String? stringInput = stdin.readLineSync();
        if(stringInput != null){
          List<String>? stringArray = stringInput.split(" ");
          if(stringArray.length == 2){
            int? x = int.tryParse(stringArray[0]);
            int? y = int.tryParse(stringArray[1]);
            if(x != null && y != null){
              int total = x * y;
              print(total);
            }
          }
        }
        count = count - 1;
      }
    }
  }
}